internal/cli/validate.go

```diff
@@ -72,6 +79,8 @@ func validateCmd() *cobra.Command {
 
 	cmd.PersistentFlags().StringVarP(&ignoredJobs, "ignored", "i", "", "set ignored jobs (comma-separated list)")
 
+	cmd.PersistentFlags().IntVar(&githubClientRetry, "client-retry", 0, "set retry count for github client")
+
```

Comment:
Need to update the action yaml with this input option as well https://github.com/clear-street/merge-gatekeeper/blob/main/action.yml